terraform/modules/frontdoor/UrlRewriterReadMe.md

```diff
@@ -0,0 +1,19 @@
+|----------------------------------------------------------------------------------------|--------------------------------------------------------------------------------------------------------|-----------------------------------------------------------------------------|
+| https://integration.register-home-to-rent.test.communities.gov.uk/dashboard            | https://integration.register-home-to-rent.test.communities.gov.uk/landlord/dashboard                   | Landlord dashboard                                                          |
+| integration.register-home-to-rent.test.communities.gov.uk/dashboard                    | integration.register-home-to-rent.test.communities.gov.uk/landlord/dashboard                           | Works whether the url scheme is included or not                             |
+| https://integration.search-landlord-home-information.test.communities.gov.uk/dashboard | https://integration.search-landlord-home-information.test.communities.gov.uk/local-authority/dashboard | Local authority dashboard                                                   |
+| https://test.search-landlord-home-information.test.communities.gov.uk/dashboard        | https://test.search-landlord-home-information.test.communities.gov.uk/local-authority/dashboard        | Works for any environment                                                   |
+| https://test.search-landlord-home-information.gov.uk/dashboard                         | https://test.search-landlord-home-information.gov.uk/local-authority/dashboard                         | Works for modified domain (needs to include .gov.uk)                        |
+| https://integration.register-home-to-rent.test.communities/dashboard                   | https://integration.register-home-to-rent.test.communities/dashboard                                   | Returns the original url if the domain does not include .gov.uk             |
+| https://integration.register-home-to-rent.test.communities/signout                     | https://integration.register-home-to-rent.test.communities/signout                                     | Returns the original url for excluded endpoints                             |
+| https://integration.register-home-to-rent.test.communities/landlord/dashboard          | https://integration.register-home-to-rent.test.communities/landlord/dashboard                          | Returns the original url if the correct service name is already included    |
+| https://integration.register-home-to-rent.test.communities/local-authority/dashboard   | https://integration.register-home-to-rent.test.communities/landlord/dashboard                          | Attempts to replace an incorrect service name with the correct service name |
```

Comment:
These last three examples would all be covered by not including `gov.uk` anyway.